services/storage/tests/unit/test_simcore_s3_dsm_utils.py

```diff
@@ -19,3 +27,48 @@
 )
 def test_compute_file_id_prefix(file_id, levels, expected):
     assert compute_file_id_prefix(file_id, levels) == expected
+
+
+_FOLDERS_PATH = Path("nested/folders/path")
+
+
+@pytest.mark.parametrize(
+    "selection, s3_object, expected",
+    [
+        (Path("single_file"), Path("single_file"), "single_file"),
+        (Path("single_folder"), Path("single_folder"), "single_folder"),
+        (_FOLDERS_PATH / "folder", _FOLDERS_PATH / "folder", "folder"),
+        (_FOLDERS_PATH / "a_file.txt", _FOLDERS_PATH / "a_file.txt", "a_file.txt"),
+        (_FOLDERS_PATH, _FOLDERS_PATH / "the/actual/path", "the/actual/path"),
+    ],
+)
+def test__strip_parent(selection: Path, s3_object: Path, expected: str):
+    assert (
+        _strip_parent(UserSelection(f"{selection}"), S3ObjectKey(f"{s3_object}"))
+        == expected
+    )
```

Comment:
so instead of using Path, now with this you need to re-create its functions... that is a bit sad.

Reply:
Ahh I see the confusion now. I'm actually trying to strip a path from another path. This means that if the `base_path="a/b/c"` and my file has a path like `file=a/b/c/d/e/f/g` the result will be `c/d/e/f/g`
Let me rename it a bit better

Reply:
yes these are functions available in Path... not in str